tests/e2e/instrumentation-apache-multicontainer/00-install-instrumentation.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: apache
+spec:
+  exporter:
+    endpoint: http://localhost:4317
+  propagators:
+    - jaeger
+    - b3
+  sampler:
+    type: parentbased_traceidratio
+    argument: "0.25"
+  apacheHttpd:
+    image: ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-apache-httpd:1.0.2
```

Comment:
isn't the default image set automatically by the operator? At least this is the case for other auto-instrumentations. 

I would prefer to use the default image, as this will get forgotten if there will be new image version